### PR TITLE
Fixes #873: Add info about News into news tab

### LINKED ISSUE #873: Add info about "News" into news tab
Google shows a basic info about how many search results are there. Please add info about the news tab in the same style as Google as follows:

* Show: "News tab shows results filtered according to news organizations. If you wish to propose or change this list, please check [here."](https://github.com/fossasia/loklak_search/blob/development/src/app/shared/news-org.ts)

screenshot from 2018-07-22 12-56-26

screenshot from 2018-07-22 12-49-06

### PR DESCRIPTION
**Pull request by simsausaurabh**
**Changes proposed in this pull request**
- Added info about no of results obtained in news.
- Added info about changing/updating the current file containing news orgs.
- Added hardcoded URL for the Github file containing the names of news orgs.
- Made the info architecture similar to Google: text-color, text-size, hover color. 

**Screenshots (if appropriate)** 

screenshot from 2018-08-04 16-27-22

**Link to live demo: http://pr-897-fossasia-loklaksearch.surge.sh**

**Closes #873**

### CODE REVIEW

File: src/app/feed/feed-news/feed-news.component.html [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<div class="message">
+  <!-- Hook up a message if needed like google does "About xxxxx results (y.yy seconds)" -->
+  News tab shows results filtered according to news organizations. If you wish to propose or change this list, please check
```
Comment (praveenojha33) [inline src/app/feed/feed-news/feed-news.component.html]: Put this message below `About xxxx results` . Lets follow a design similar to market leader.

File: src/app/feed/feed-news/feed-news.component.html [hunk 2]
```diff
@@ -1,3 +1,11 @@
+<div class="message">
+  <!-- Hook up a message if needed like google does "About xxxxx results (y.yy seconds)" -->
```
Comment (praveenojha33) [inline src/app/feed/feed-news/feed-news.component.html]: Please make comment simple like `<!-- Message about total results-->` we are doing it for users' experience. No need to mention who does that and who does not.

File: src/app/feed/feed-news/feed-news.component.html (hunk 1 above)
Comment (simsausaurabh) [reply src/app/feed/feed-news/feed-news.component.html]: Made the changes.

**Comment by simsausaurabh** [comment]
@praveenojha33 Made the changes, please review again.

**Comment by simsausaurabh** [comment]
@praveenojha33 Please review, I have updated the changes.

screenshot from 2018-08-06 19-59-19

**Pull request merged**